src/main/java/kernbeisser/Windows/Inventory/InventoryController.java

```diff
@@ -22,65 +12,80 @@
 import kernbeisser.Reports.*;
 import kernbeisser.Security.Key;
 import kernbeisser.Useful.CSV;
+import kernbeisser.Useful.Date;
 import kernbeisser.Useful.Tools;
 import kernbeisser.Windows.Inventory.Counting.CountingController;
 import kernbeisser.Windows.MVC.Controller;
 import kernbeisser.Windows.MVC.Linked;
 import lombok.Getter;
 import lombok.SneakyThrows;
 
+import java.io.File;
+import java.time.LocalDate;
+import java.time.chrono.ChronoLocalDate;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static javax.swing.SwingConstants.RIGHT;
+
 public class InventoryController extends Controller<InventoryView, InventoryModel> {
-  @Linked @Getter private final ObjectViewController<Shelf> shelfViewController;
+  @Linked
+  @Getter
+  private final ObjectViewController<Shelf> shelfViewController;
+  Setting inventoryScheduledDate = Setting.INVENTORY_SCHEDULED_DATE;
```

Comment:
besser private final?

Reply:
ich habe die Variable deshalb eingeführt und *nicht* mit "private final" belegt, damit die in der Testklasse "gemockt" werden kann. 
Aus reinen Modellierung stimme ich Dir zu, aber im Sinne der Testbarkeit nicht :)